modules/fbx/fbx_document.cpp

```diff
@@ -2461,10 +2490,12 @@ PackedByteArray FBXDocument::generate_buffer(Ref<GLTFState> p_state) {
 }
 
 Error FBXDocument::write_to_filesystem(Ref<GLTFState> p_state, const String &p_path) {
+	//todo: everything
```

Comment:
```suggestion
	//todo: everything
```

I don't know when this todo will ever be done, so it'll probably stay here forever. Would like to delete it.

Reply:
@bqqbarbhg doesn't have near plans for making a fbx exporter